content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -0,0 +1,300 @@
+---
+title: 用户名字空间
+reviewers:
+content_type: concept
+weight: 160
+min-kubernetes-server-version: v1.25
+---
+<!--
+title: User Namespaces
+reviewers:
+content_type: concept
+weight: 160
+min-kubernetes-server-version: v1.25
+-->
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+<!--
+This page explains how user namespaces are used in Kubernetes pods. A user
+namespace allows to isolate the user running inside the container from the one
+in the host.
+-->
+本页展示如何在 Kubernetes Pod 中使用**用户名字空间（User Namespaces）**。
+用户名字空间可以将容器内的用户与主机上的用户隔离开来。
+
+<!--
+A process running as root in a container can run as a different (non-root) user
+in the host; in other words, the process has full privileges for operations
+inside the user namespace, but is unprivileged for operations outside the
+namespace.
+-->
+一个以 root 用户运行的进程的容器，这个容器可以由宿主机中不同的用户（非 root）运行，
+进程在用户名字空间内部拥有执行操作的全部特权，但在用户名字空间外部并没有执行操作的特权。
+
+<!--
+You can use this feature to reduce the damage a compromised container can do to
+the host or other pods in the same node. There are [several security
+vulnerabilities][KEP-vulns] rated either **HIGH** or **CRITICAL** that were not
+exploitable when user namespaces is active. It is expected user namespace will
+mitigate some future vulnerabilities too.
+-->
+你可以使用这个特性来减少有危害的容器对同一宿主机上其他容器的影响。
+当用户名字空间被启用后，一些被评为 **HIGH** 或者 **CRITICAL** 的[安全漏洞][KEP-vulns]是无法被利用的。
+相信用户名字空间也能减轻一些未来的漏洞的影响。
+
+[KEP-vulns]: https://github.com/kubernetes/enhancements/tree/217d790720c5aef09b8bd4d6ca96284a0affe6c2/keps/sig-node/127-user-namespaces#motivation
+
+<!-- body -->
+## {{% heading "prerequisites" %}}
+
+{{% thirdparty-content single="true" %}}
+<!-- if adding another runtime in the future, omit the single setting -->
+
+<!--
+This is a Linux only feature. In addition, support is needed in the 
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+to use this feature with Kubernetes stateless pods:
+
+* CRI-O: v1.25 has support for user namespaces.
+
+* containerd: support is planned for the 1.7 release. See containerd
+  issue [#7063][containerd-userns-issue] for more details.
+
+Support for this in [cri-dockerd is not planned][CRI-dockerd-issue] yet.
+-->
+这是Linux独有的功能。
+除此之外, 还需要{{< glossary_tooltip text="容器运行时" term_id="container-runtime" >}}
+提供相应的支持，才能将此特性与 Kubernetes 无状态 Pod 一起使用：
+
+* CRI-O：v1.25 已支持用户名字空间。
+
+* Containerd：计划在 1.7 版本中提供支持。
```

Comment:
```suggestion
* containerd：计划在 1.7 版本中提供支持。
```

Reply:
yeah, it should be lowercase, thank you Tim!